server.js

```diff
@@ -140,12 +154,9 @@ app.post('/push-build/:tag/:platform', handleAsync(async function (req, res) {
 	console.log(`Track: ${branch} => ${track} (${tracks[track]}) [enabled: ${!!enabledTracks[track]}]`);
 
 	if (!enabledTracks[track]) {
-		throw new Error(`Track not enabled: ${track}`);
+		throw boom.accepted(`Track not enabled: ${track}`);
```

Comment:
That's not an error, right? Why using `boom` as such? You could just use `res` to send an early response.

Reply:
It is an error, but I want to return a specific status code in such case. Also I want to separate validation&logic from dealing with `req/res` directly.